cli/tests/053_import_compression/main.ts

```diff
@@ -0,0 +1,2 @@
+import "http://127.0.0.1:4545/cli/tests/053_import_compression/gziped";
+import "http://127.0.0.1:4545/cli/tests/053_import_compression/brotli";
```

Comment:
Please add test cases for `fetch` API:
```
const resp1 = await fetch("http://127.0.0.1:4545/cli/tests/053_import_compression/gziped");
const resp2 = await fetch("http://127.0.0.1:4545/cli/tests/053_import_compression/brotli");
```
I think right now `fetch` API will happily accept gzip/brotli and then return garbage when you try to read the response

Reply:
Sorry, I tried to make the brotli decoding effective in fetch ops, but I don't understand async read at present, I don't know how to start

Reply:
You need to look at `cli/http_util.rs::HttpBody` it's a wrapper around `reqwest`'s Response that reads body in chunks - it looks like you're using `gzip` provided out-of-the-box by `reqwest` so I suspect it might be working out of the box as well, but `brotli` is still not handled. Start by adding tests and we can figure the rest later

Reply:
I added gzip testing for fetch, but brotli support, I need to learn more about async_read